lms/djangoapps/edraak_university/views.py

```diff
@@ -240,9 +245,18 @@ def form_valid(self, form):
 class UniversityIDUpdateView(ContextMixin, generic.UpdateView):
     model = UniversityID
     template_name = 'edraak_university/instructor/update.html'
+    form_class = UniversityIDForm
```

Comment:
This was a separate view in the start. But due to needing it to be tied to the model to retrieve the correct queryset me and @ahmedaljazzar decided to use the same view.